module/input/SensorFilter/src/SensorFilter.hpp

```diff
@@ -107,8 +107,8 @@ namespace module::input {
         /// @brief tinyrobotics model of NUgus used for kinematics
         tinyrobotics::Model<double, n_servos> nugus_model;
 
-        /// @brief Current walk phase of the robot
-        WalkState::Phase current_walk_phase = WalkState::Phase::LEFT;
+        /// @brief Current planted foot phase of the robot
+        WalkState::Phase current_planted_foot_phase = WalkState::Phase::LEFT;
```

Comment:
It might make more sense to make this `WalkState::Phase::DOUBLE` and consider changing to `WalkState::Phase::FEET_DOWN` for more expressiveness as the robot would, by default, be standing on its feet during stand I assume

Reply:
This variable is used for tracking and updating the anchor frame, and should never be double, I have renamed things and added comments to try make it clearer, there is probably a nicer way to track and name all of this ...